src/main.rs

```diff
@@ -143,6 +146,16 @@ fn main() {
             },
           }
 
+          let parent_height = ui.rect_of(ids.settop2).unwrap().h();
```

Comment:
This is a hack and is completely unneeded. I've already explained at least 3 times on IRC how to fix it properly.